CONTRIBUTING.rst

```diff
@@ -1,6 +1,22 @@
 Contributor Guidelines
 ======================
 
+How to contribute
+-----------------
+We love contributions! SkyPy is open source,
+built on open source, and we'd love to have you hang out in our community.
+Whether you would like to contribute to SkyPy with your own piece of code or
+helping SkyPy develop a concrete feature of the SkyPy roadmap:
```

Comment:
Is 'SkyPy' roadmap a specific document? It reads here as if it is, which the reader does not have access to.

You could just replace 'SkyPy roadmap' with 'SkyPy' throughout.

Reply:
> Is 'SkyPy' roadmap a specific document? It reads here as if it is, which the reader does not have access to.
> 
> You could just replace 'SkyPy roadmap' with 'SkyPy' throughout.

agreed